.github/workflows/conflict-labeler.yml

```diff
@@ -0,0 +1,26 @@
+name: Merge Conflict Labeler
+
+on:
+  push:
+    branches:
+      - master
+  pull_request_target:
+    branches:
+      - master
+    types: [synchronize]
+
+jobs:
+  label:
+    name: Labeling
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'openbullet/OpenBullet2' }}
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Apply label
+        uses: eps1lon/actions-label-merge-conflict@v3
+        with:
+          dirtyLabel: 'merge-conflict'
+          commentOnDirty: 'This pull request has merge conflicts. Please resolve the conflicts so the PR can be successfully reviewed and merged.'
+          repoToken: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
There is no such secret at the moment in the repo, is it something that is automatically defined or do I have to create a fine-grained token? Do you have more information on this?

Reply:
1. To follow your previous comment should I remove the publish.yaml ?
2. The token is automatically created by Github, no need to setup anything more than just the `merge-conflict` label